### PR TITLE
[12.0][IMP] stock_picking_batch_extended: UX improvements

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT21152

### CODE REVIEW

File: stock_picking_batch_extended/models/stock_batch_picking.py [hunk 1]
```diff
@@ -130,6 +135,12 @@ def _compute_entire_package_ids(self):
                         'entire_package_detail_ids'),
                 })
 
+    def _compute_picking_count(self):
+        """Calculate number of pickings."""
+        for batch in self:
+            batch.picking_count = self.env['stock.picking'].search_count(
+                [('batch_id', '=', batch.id)])
```
Comment (Tardo) [inline stock_picking_batch_extended/models/stock_batch_picking.py]: Can improve performance using read_group...
```python
datas = self.env['stock.picking'].read_group(domain=[('batch_id', 'in', self.ids)], fields='batch_id', groupby='batch_id')
counts = {data['batch_id'][0]: data['batch_id_count'] for data in datas}
for batch in self:
    batch.picking_count = counts.get(batch.id, 0)
```

**Review by Tardo — CHANGES_REQUESTED** [review]

File: stock_picking_batch_extended/models/stock_batch_picking.py (hunk 1 above)
Comment (carlosdauden) [reply stock_picking_batch_extended/models/stock_batch_picking.py]: Changes done

**Review by Tardo — APPROVED** [review]
:+1: LGTM

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**